samples/ecom/src/main/java/com/kody/ExampleNewPayment.java

```diff
@@ -46,6 +46,17 @@ public static void initiateEcomPayment(String storeId, String orderId, String pa
                 .setCurrency(currencyCode)
                 .setOrderId(orderId)
                 .setReturnUrl(returnUrl)
+                .setCaptureOptions(PaymentInitiationRequest.CaptureOptions.newBuilder()
+                        .setCaptureSettings(PaymentInitiationRequest.CaptureOptions.CaptureSettings.newBuilder()
+                                .setDelayedCapture(true)
```

Comment:
The basic examples should use the defaults. Please create an alternative example to that sets these to true